src/algoliasearch.helper.js

```diff
@@ -366,22 +366,33 @@ AlgoliaSearchHelper.prototype.nextPage = function() {
  * @return {AlgoliaSearchHelper}
  */
 AlgoliaSearchHelper.prototype.previousPage = function() {
-  return this.setCurrentPage(this.state.page - 1);
+  return this.setPage(this.state.page - 1);
 };
 
+function setCurrentPage(page) {
+  if (page < 0) throw new Error('Page requested below 0.');
```

Comment:
Well that's the current behavior and the idea behind when implemented was "best do it before than later"